composer.json

```diff
@@ -17,7 +17,7 @@
         "ckeditor/ckeditor": "4.6.2",
         "coreui/coreui": "^2.1",
         "dmhendricks/file-icon-vectors": "~1.0",
-        "doctrine/dbal": "~2.9.0",
+        "doctrine/dbal": "~2.9",
```

Comment:
this will result in composer update automatically pulling every minor version. I would have assumed, that we only want to update patch level updates?

Reply:
This is how it was in previous versions. I guess this should be ok, but you may convince me otherwise.

Reply:
(That's why it can have been 2.12 as noted in the ticket https://github.com/chameleon-system/chameleon-system/issues/670)

Reply:
ok - I had a look hat the dbal planing. 2.x should remain backward compatible. So I guess ~2.9 is ok. However, I would suggest that we require ~2.12 since it contains a lot of changes that aid in preparing an upgrade to 3.0 later. That way we can start to prepare for the 3.0 upgrade.